Userland/Libraries/LibWeb/DOM/DOMTokenList.cpp

```diff
@@ -68,8 +68,8 @@ DOMTokenList::DOMTokenList(Element& associated_element, FlyString associated_att
 {
     m_legacy_platform_object_flags = LegacyPlatformObjectFlags { .supports_indexed_properties = 1 };
 
-    auto value = associated_element.deprecated_get_attribute(m_associated_attribute);
-    associated_attribute_changed(value);
+    if (auto value = associated_element.get_attribute(m_associated_attribute); value.has_value())
+        associated_attribute_changed(*value);
```

Comment:
eh, but this one is still wrong. We should always call `associated_attribute_changed` here in DOMTokenList, just with an empty string. That's what the spec references I commented said.